async_task.js

```diff
@@ -0,0 +1,97 @@
+/* 1. Створити модуль api_helper.js, в якому будуть реалізовані функції посилання get та post запитів
+та десеріалізація респонсу у json об'єкт, який вони будуть повертати (при реалізації поста зверніть увагу на хедери, 
+які вказані у інструкції під блоком Post with JSON). Для створення запитів використати пакет
+node-fetch, інструкця по використанню тут https://www.npmjs.com/package/node-fetch?activeTab=readme, для 
+реалізації асинхронності використати async/await */
+
+import * as request from "./api_helper/api_helper.js";
+
+const baseUrl = "https://jsonplaceholder.typicode.com/posts/";
+// Приклад використання GET
+const getData = async () => {
```

Comment:
```suggestion
const getData = async (baseUrl) => {
```
універсальніше буде передавати всі параметри ззовні щоб мати цю функцію перевикористовуваною